code/_onclick/hud/alert.dm

```diff
@@ -315,6 +315,20 @@ Recharging stations are available in robotics, the dormitory bathrooms, and the
 	desc = "Unit's power cell is running low. Recharging stations are available in robotics, the dormitory bathrooms, and the AI satellite."
 	icon_state = "lowcell"
 
+//Diona Nymph
+/obj/screen/alert/nymph
+	name = "Gestalt merge"
+	desc = "You have merged with a diona gestalt and are now part of it's biomass. You can still wiggle yourself free though."
+	icon_state = "nymph"
+
+/obj/screen/alert/nymph/Click()
+	if(!usr || !usr.client)
+		return
+	else
+		if(istype(usr, /mob/living/simple_animal/diona))
```

Comment:
This should be `isnymph`

Additionally, the `else` and the `if` lines should be on the same line